code/drasil-metadata/lib/Data/Drasil/DrasilConfig.hs

```diff
@@ -10,10 +10,22 @@ import Utils.Drasil(drasilConfigPath)
 
 newtype DrasilConfig = DrasilConfig {version :: String} deriving Generic
 
+-- The number of folders that needs to be exited in order to obtain the
+-- DrasilConfiguration.json file
+defaultFolderVal = 3
+
 instance FromJSON DrasilConfig
 
-obtainVersion :: Int -> IO String
-obtainVersion num = do
+-- Reads the DrasilConfiguration.json file and extracts the data from that file.
+obtainVersion :: Maybe Int -> IO String
+obtainVersion Nothing = do
+  input <- B.readFile $ drasilConfigPath defaultFolderVal
```

Comment:
Instead of pattern matching, using [`fromMaybe`](https://hackage.haskell.org/package/base-4.18.0.0/docs/Data-Maybe.html#v:fromMaybe):

```suggestion
obtainVersion num = do
  input <- B.readFile $ drasilConfigPath $ fromMaybe defaultFolderVal num
```

will let you get rid of the `obtainVersion (Just num)` case below, reducing duplication

Reply:
Thanks for that suggestion! I did not know that was a possibility!